javers-core/src/main/java/org/javers/core/graph/ObjectNode.java

```diff
@@ -75,7 +78,11 @@ public GlobalId getReference(Property property){
 
     public Object getPropertyValue(Property property) {
         Validate.argumentIsNotNull(property);
-        return cdo.getPropertyValue(property);
+        try {
+            return cdo.getPropertyValue(property);
+        } catch (JaversGetterException e){
```

Comment:
Wolałbym przekazać ten getter exception, wtedy jeżeli zdarzyłoby się coś innego dziwnego to masz piękny komunikat z JaversGetterExcepion:

```
JaversException: TYPE_MISMATCH JaVers runtime error - Looks like you are comparing two objects with different types. There is no property 'b0' in type 'org.javers.core.cases.TypeMismatchExceptionClasses$B1'.
	at org.javers.core.graph.ObjectNode.getPropertyValue(ObjectNode.java:84)
	at org.javers.core.diff.RealNodePair.getRightPropertyValue(RealNodePair.java:40)
	at org.javers.core.diff.appenders.ValueChangeAppender.calculateChanges(ValueChangeAppender.java:26)
	at org.javers.core.diff.appenders.ValueChangeAppender.calculateChanges(ValueChangeAppender.java:13)
	at org.javers.core.diff.DiffFactory.appendChanges(DiffFactory.java:142)
	at org.javers.core.diff.DiffFactory.appendPropertyChanges(DiffFactory.java:132)
	at org.javers.core.diff.DiffFactory.createAndAppendChanges(DiffFactory.java:114)
	at org.javers.core.diff.DiffFactory.create(DiffFactory.java:65)
	at org.javers.core.diff.DiffFactory.compare(DiffFactory.java:58)
	at org.javers.core.JaversCore.compare(JaversCore.java:75)
	at org.javers.core.cases.TypeMismatchExceptionIssue.should throw proper exception when comparing different types(TypeMismatchExceptionIssue.groovy:20)
Caused by: org.javers.common.exception.JaversGetterException: error getting value from the field {Field class java.lang.String b0; //declaring class:org.javers.core.cases.TypeMismatchExceptionClasses$B0}, java.lang.IllegalArgumentException: Can not set java.lang.String field org.javers.core.cases.TypeMismatchExceptionClasses$B0.b0 to org.javers.core.cases.TypeMismatchExceptionClasses$B1
	at org.javers.common.reflection.JaversField.invokeEvenIfPrivate(JaversField.java:34)
	at org.javers.core.metamodel.property.Property.get(Property.java:43)
	at org.javers.core.metamodel.object.CdoWrapper.getPropertyValue(CdoWrapper.java:35)
	at org.javers.core.graph.ObjectNode.getPropertyValue(ObjectNode.java:82)
	... 10 more
```

vs

```
JaversException: TYPE_MISMATCH JaVers runtime error - Looks like you are comparing two objects with different types. There is no property 'b0' in type 'org.javers.core.cases.TypeMismatchExceptionClasses$B1'.
	at org.javers.core.graph.ObjectNode.getPropertyValue(ObjectNode.java:84)
	at org.javers.core.diff.RealNodePair.getRightPropertyValue(RealNodePair.java:40)
	at org.javers.core.diff.appenders.ValueChangeAppender.calculateChanges(ValueChangeAppender.java:26)
	at org.javers.core.diff.appenders.ValueChangeAppender.calculateChanges(ValueChangeAppender.java:13)
	at org.javers.core.diff.DiffFactory.appendChanges(DiffFactory.java:142)
	at org.javers.core.diff.DiffFactory.appendPropertyChanges(DiffFactory.java:132)
	at org.javers.core.diff.DiffFactory.createAndAppendChanges(DiffFactory.java:114)
	at org.javers.core.diff.DiffFactory.create(DiffFactory.java:65)
	at org.javers.core.diff.DiffFactory.compare(DiffFactory.java:58)
	at org.javers.core.JaversCore.compare(JaversCore.java:75)
	at org.javers.core.cases.TypeMismatchExceptionIssue.should throw proper exception when comparing different types(TypeMismatchExceptionIssue.groovy:20)
```

w tej chwili nie ma takiej konstrukcji w JaversException ale to chyba nie problem